src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -175,6 +184,11 @@ private String unescapeChars(String escapedString) {
     return unescapedString;
   }
 
+  private boolean isWildcardPattern(String inputString) {
```

Comment:
This function wraps around the current isWildcardPatternStr() function. It returns false whenever the schema is being used from the session because this schema should not be treated as a wildcard pattern.